README.md

```diff
@@ -318,6 +318,34 @@ comments within the function for details.
 `BestRecordStructureFromPath` accepts the same optional arguments as
 `BestRecordStructure`.
 
+### Report
+
+A report is generated based on the output of `Profile()`. The report is accessible via a Workunit's *Resources* tab within ECL Watch. For example:
+![Screen capture displaying active Resources tab](https://user-images.githubusercontent.com/1891935/57020403-2ac29480-6bf7-11e9-9584-a6fd23a3b4c4.png)
+
+A report can also be viewed directly once a workunit has completed. For example:
+https://play.hpccsystems.com:18010/WsWorkunits/res/W20190430-175856/report/res/index.html
+(NOTE: This example URL was valid at the time of writing this README entry. Simply swap out the WUID in the URL with the WUID containing your Profile results to view your report)
+
+Every `attribute` in the Profile result is represented by a row of information. 
+Each row of information is organized into several columns. Here is a short description 
+of each column:
+
+1. Type information, Cardinality Count & Filled Count
+2. Min, Avg, Max Length (for string attributes) or Mean, Std. Deviation, Quartiles (for numeric attributes)
+3. Quartile bell curve and candlestick
+    * only shown for attributes with `is_numeric` === `true`
+    * this column is omitted if the above condition fails for all attributes
+4. Cardinality Breakdown listed by count descending
+    * only shown for attributes with `cardinality_breakdown` content
+    * this column is omitted if the above condition fails for all attributes
+5. Popular Patterns 
+    * only shown for attributes with `popular_patterns` content
+    * this column is omitted if the above condition fails for all attributes
+
+This is a screen capture displaying a report row for a string attribute ("Test_Name") and a numeric attribute ("Test_Score"):
```

Comment:
Some versions of Markdown want to italicize parts of this text (they interpret the underlines as an italics start).  Please escape the underlines (e.g. "Test\_Name").

Reply:
Also, please insert a blank line so as to separate the image from the text a bit.